src/views/CaseDetail.vue

```diff
@@ -0,0 +1,246 @@
+      pageId: null,
+    }
+  },
+  created() {
+    this.pageId = (this.id) ? this.id : localStorage.getItem('caseId');
+    if (!this.pageId) {
+      this.$router.push({ name: 'home' });
+    }
+    API.cases.getById(this.pageId).then((doc) => {
+      if (doc.exists) {
+        this.details = {
+          category: doc.data().feedbackType,
+          id: this.id,
+          ...doc.data(),
+        };
+      } else {
+        this.removeItem('caseId');
+        this.$router.push({ name: 'home' });
+      }
+    })
+    .catch((err) => {
+      console.warn(err)
+    });
+  },
+  methods: {
+    convertDate(date) {
+      return utilities.dateMapper(date)
+    },
+    camelCaseToNormal(data) {
+      return data.replace(/([A-Z])/g, ' $1')
+      .replace(/^./, function(str){ return str.toUpperCase(); })
+    },
+    hideElements(target, stringArray) {
+      var value = 0;
+      stringArray.forEach(function(word){
+        value = value + target.includes(word);
+      });
+      return (value === 1);
+    },
+    copyToClipboard() {
+      var copyText = this.$refs.passcode;
+      copyText.select();
+      copyText.setSelectionRange(0, 99999);
+      document.execCommand("copy");
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.container {
+  max-width: 1140px;
+  margin: 0 auto;
+  padding-top: 42px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.case-detail__content {
+  display: flex;
+  background: white;
+  flex-wrap: wrap;
+}
+
+.case-detail__sidebar {
+  flex: 0 0 48%;
+  margin: 0 2% 0 0%;
+}
+
+.case-detail__wrapper {
+  height: 600px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: scroll;
+}
+
+.case-detail__object {
+  border: 1px solid var(--light-gray);
+  border-left: none;
+  border-top: none;
+  margin: 0;
+  padding: 12px 20px;
+  font-size: 16px;
+  max-width: 100%;
+  flex: 0 0 100%;
+  box-sizing: border-box;
+
+  &:last-child {
+    border-top: 1px solid var(--light-gray);
+    margin-top: -1px;
+    width: 100%;
+  }
+
+  b {
+    display: block;
+    font-weight: 500;
+    opacity: .6;
+    font-size: 14px;
+  }
+}
+
+.case-detail__object--half {
+  max-width: 50%;
+  flex: 0 0 50%;
+}
+
+.case-detail__object--noAnswer {
+  background-image: linear-gradient(rgba(223, 223, 223, 0.04) .1em, transparent .1em), linear-gradient(90deg, rgba(223, 223, 223, 0.04) .1em, transparent .1em);
+  background-size: .3em .3em;
+  overflow: hidden;
+  opacity: .6;
+
+  span {
+    font-size: 11px;
+    text-transform: uppercase;
+    font-weight: 700;
+    letter-spacing: .3px;
+    opacity: .2;
+  }
+}
+
+.case-detail__object--hide {
+  visibility: hidden;
+  display: none;
+}
+
+.passcode {
+  display: flex;
+  flex: 0 0 100%;
+  flex-direction: row;
+  align-items: center;
+
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none; 
+
+  input {
+    background: #bbddcd;
+    padding: 0 12px;
+    color: #000000;
```

Comment:
I'm not sure, but I think this will be more consistent throughout the app.

```suggestion
    color: var(--dark-black);
```

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/f9e6d2b42b6328e0e714a6ff24c0c25b67617266